airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -336,14 +346,10 @@ def get_json_schema(self) -> Mapping[str, Any]:
             "activityDate": {"type": ["null", "string"], "format": "date-time"},
             "activityTypeId": {"type": ["null", "integer"]},
             "campaignId": {"type": ["null", "integer"]},
-            "attributes": {"type": ["null", "string"]},
+            "primaryAttributeValueId": {"type": ["null", "string"]},
+            "primaryAttributeValue": {"type": ["null", "string"]},
```

Comment:
Need add `primaryAttributeName`

Reply:
I have dumped the header of all 'activities_*' streams and found that:

1) it's the same for all steams
2) it consists of next fields:

```
marketoGUID,
leadId,
activityDate,
activityTypeId,
campaignId,
primaryAttributeValueId,
primaryAttributeValue,
attributes
```

I as you can see there is no `primaryAttributeName` field 
I have also not found any references of this field in other places like this
https://developers.marketo.com/rest-api/lead-database/activities/

I decided to remove.

What do you think ?

Reply:
Ok. We can remove it.